src/taxonomy/taxonomy-card/TaxonomyCard.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+import {
+  Badge,
+  Card,
+  OverlayTrigger,
+  Popover,
+} from '@edx/paragon';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import messages from '../messages';
+import TaxonomyCardMenu from './TaxonomyCardMenu';
+import ExportModal from '../modals/ExportModal';
+
+const TaxonomyCard = ({ className, original, intl }) => {
+  const {
+    id, name, description, systemDefined, orgsCount,
+  } = original;
+
+  const [isExportModalOpen, setIsExportModalOpen] = useState(false);
+
+  const orgsCountEnabled = () => orgsCount !== undefined && orgsCount !== 0;
+
+  const onClickMenuItem = (menuName) => {
+    switch (menuName) {
+    case 'export':
+      setIsExportModalOpen(true);
+      break;
+    default:
+      /* istanbul ignore next */
+      break;
+    }
+  };
+
+  const getSystemBadgeToolTip = () => (
+    <Popover id={`system-defined-tooltip-${id}`}>
+      <Popover.Title as="h5">
+        {intl.formatMessage(messages.systemTaxonomyPopoverTitle)}
+      </Popover.Title>
+      <Popover.Content>
+        {intl.formatMessage(messages.systemTaxonomyPopoverBody)}
+      </Popover.Content>
+    </Popover>
+  );
```

Comment:
Instead of having this as a function, you can make it a secondary component in this file? i.e. move it out of this function and refactor it as follows:

```jsx
  const SystemBadgeToolTip = ({id, intl}) => (
    <Popover id={`system-defined-tooltip-${id}`}>
      <Popover.Title as="h5">
        {intl.formatMessage(messages.systemTaxonomyPopoverTitle)}
      </Popover.Title>
      <Popover.Content>
        {intl.formatMessage(messages.systemTaxonomyPopoverBody)}
      </Popover.Content>
    </Popover>
  );
```

Likwise with HeaderSubtitle

Reply:
I updated this here https://github.com/openedx/frontend-app-course-authoring/pull/645/commits/a21fac2c7f4d724e5c73323aa6e19d70a1f902cb
I have only been able to extract the `HeaderSubtitle`. When extracting `SystemBadgeToolTip` UI errors occur and the tooltip is not displayed correctly.